pkg/icingadb/runtime_updates.go

```diff
@@ -301,17 +301,19 @@ func (r *RuntimeUpdates) xRead(ctx context.Context, updateMessagesByKey map[stri
 // those messages into Icinga DB entities (contracts.Entity) using the provided structifier.
 // Converted entities are inserted into the upsertEntities or deleteIds channel depending on the "runtime_type" message field.
 func structifyStream(
-	ctx context.Context, updateMessages <-chan redis.XMessage, upsertEntities, upserted chan database.Entity,
-	deleteIds, deleted chan interface{}, structifier structify.MapStructifier,
+	ctx context.Context, updateMessages <-chan redis.XMessage, upsertEntities chan<- database.Entity,
+	upserted <-chan database.Entity, deleteIds chan<- any, deleted <-chan any, structifier structify.MapStructifier,
```

Comment:
The parameter list got quite long and is imo hard to read in its current format. Would you mind reformatting it, having one parameter each line.